src/git/github.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2022 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const Git = require('./git');
+const parsePath = require('parse-filepath');
+
+module.exports = class Github extends Git {
+  constructor(reqOpt) {
+    super(reqOpt);  
+  }
+
+  getReqUrl() {
+    let enterprise = '';
+    if (this.host == 'github.com') {
+      enterprise = 'https://api.github.com';
+    } else {
+      enterprise = `http://${this.host}/api/v3`;
+    }
+    if (this.path.endsWith('/')) {
+      this.path = this.path.slice(0, -1);
+    }
```

Comment:
Seems like it would be a problem for the path to end in `/` for gitlab as well. Suggest stripping any trailing slash in the Git constructor at the beginning, so all subclasses have a form they can rely on.